src/style.css

```diff
@@ -0,0 +1,117 @@
+
+/* @font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 300;
+} */
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+  
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+body {
+  --bigsearch-height: 70px;
+  --smallsearch-height: 42px;
+  --smalllogo-height: 11px;
+  --biglogo-height: 19px;
+}
+
+.container {
+  position: relative;
+  margin-top: 20px;
+}
+
+#big_form,
+#small_form {
```

Comment:
You don't need to use #. All you need is to call needed html elemet by its class.